### PR TITLE
Move SendToWebtech view into this project

### PR DESCRIPTION
**Pull request by kand**
New `BugReportEmail` view will replace `SendToWebtech` views in each site.

### CODE REVIEW

**Comment by kand** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;LuHpzeli&#x2F;174-update-cms-bug-ticket-email-address"> Update CMS bug ticket email address</a>

**Comment by kand** [comment]
@MichaelButkovic @benghaziboy @mparent61

File: bulbs/api/urls.py [hunk 1]
```diff
@@ -2,10 +2,11 @@
 from django.conf.urls import url, include
 
 from bulbs.cms_notifications.api import notifications_view
-from .views import api_v1_router, MeViewSet
+from .views import api_v1_router, MeViewSet, BugReportEmail
 
 
 urlpatterns = (
+    url(r"^report-bug/?$", BugReportEmail.as_view(), name="report-bug"),
```
Comment (mparent61) [inline bulbs/api/urls.py]: I'd keep name ordering consistent, either "bug-report/BugReportEmail" or "report-bug/ReportBugEmail"

File: scripts/test [hunk 2]
```diff
@@ -1,5 +1,9 @@
 #!/bin/sh
-# Run all tests
-#
+# Summary: Run all tests. Requires that your /etc/hosts file has an entry for
+#   your docker machine with the name `dockerhost`.
 
-py.test "$@"
+# use parens here to prevent exporting variable to parent env
+(
+  export ELASTICSEARCH_HOST=dockerhost
```
Comment (mparent61) [inline scripts/test]: I don't think we should require Docker for this non-docker project. Instead developers can optionally set the ELASTICSEARCH_HOST env variable to any hostname that points to their docker VM.

**Comment by mparent61** [comment]
LGTM

File: scripts/test [hunk 3]
```diff
@@ -1,5 +1,13 @@
...
+# Summary: Run all tests. Requires that you have an es instance running locally
+#   or are pointing to one with an env var ELASTICSEARCH_HOST.
 
-py.test "$@"
+(
+  # use parens here to prevent exporting variable to parent env
+  if [ -z "$ELASTICSEARCH_HOST" ]; then
+    # default to localhost if this var isn't already set
+    export ELASTICSEARCH_HOST=localhost
+  fi
```
Comment (kand) [inline scripts/test]: @mparent61 is this more along the lines of what you were thinking?

Comment (mparent61) [inline scripts/test]: Yeah, but you don't even have to add this line. The code already checks for a "ELASTICSEARCH_HOST" environment variable, else defaults to "localhost".

https://github.com/theonion/django-bulbs/blob/master/example/settings.py#L103

Just add this to your `~/.bashrc`, `~/.bash_profile`, etc.:

`export ELASTICSEARCH_HOST=dockerhost`

**Comment by kand** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;9jJD9aSI&#x2F;255-move-bug-reporter-views-to-django-bulbs"> Move bug reporter views to django-bulbs</a>

**Pull request merged**